.vscode/settings.json

```diff
@@ -1,12 +1,11 @@
 {
   "python.defaultInterpreterPath": ".venv/bin/python",
-  "autoDocstring.docstringFormat": "google",
```

Comment:
Shouldn't we keep this?

Reply:
I stopped using it and just use copilot so it showed not used but yes can keep probably better.